src/Discord/Parts/Thread/Thread.php

```diff
@@ -476,7 +479,7 @@ public function getPinnedMessages(): ExtendedPromiseInterface
      *
      * @link https://discord.com/developers/docs/resources/channel#bulk-delete-messages
      *
-     * @param array       $messages
+     * @param array|Traversable $messages An array of messages to delete.
      * @param string|null $reason   Reason for Audit Log (only for bulk messages).
```

Comment:
```suggestion
     * @param array|Traversable $messages An array of messages to delete.
     * @param string|null       $reason   Reason for Audit Log (only for bulk messages).
```
Running the linting should also fix this